src/components/LoginMenu.vue

```diff
@@ -105,7 +105,14 @@ export default {
   },
 
   methods: {
-    logout: () => store.dispatch.logout(),
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    async logout(this: any) {
+      // Perform the logout action, then redirect the user to the home page.
+      // This is to prevent the logged-out user from continuing to look at, e.g.,
+      // workspaces or tables they may have been viewing at the time of logout.
+      await store.dispatch.logout();
+      this.$router.push('home');
```

Comment:
If you're already on the home page, this causes an error in the console with `vue-router`, because you're trying to navigate to the route that you're already at. I'd think refreshing the page here would be a good idea, to both ensure that new data is loaded, and that old data doesn't persist. I believe `this.$router.go()` would accomplish this.

Reply:
I'd prefer not to refresh at this point, since it would feel disruptive. I will look for a way to avoid the console error.

Reply:
See 3d82367686ab8bae95335afba8f39ef1473fd5b3.